string name
---
geometry_msgs/PointStamped pointOut
float32 theta
float32 l

